docker/master/Dockerfile

```diff
@@ -54,20 +55,22 @@ RUN cd /infer && \
     ./configure && \
     ./facebook-clang-plugins/clang/setup.sh
 
-# Hackish for now: pull to get the latest version
-RUN cd /infer && git pull 
+# Generate a release
+RUN cd /infer && make install-with-libs BUILD_MODE=opt PATCHELF=patchelf DESTDIR="/infer-release" libdir_relative_to_bindir="../lib"
+
+FROM debian:stretch-slim AS executor 
+
+# Install python 2.7 since infer requires it to run
+RUN apt-get update && apt-get install --yes --no-install-recommends \
+    python2.7
```

Comment:
Why is `python2.7` needed here but none of the other dependencies from the compilator are? I'm surprised because some of them are also runtime dependencies, like `libgmp` and `libmpfr`. Or perhaps they're installed by default, only their `-dev` counterparts aren't?

Reply:
When generating the "release" version
 https://github.com/facebook/infer/blob/8ae6b1faf3d72d0bbafe352a3c9e15bc62e9d733/docker/master/Dockerfile#L59-L64
dependencies are moved to the "lib" folder (inside the installation path of Infer) and the binaries are patched in order to use them. But for any reason that is not made with python, so if you move the release version to another container without python2.7, it won't work.